From a0ea0c0bff2b9c01d4cfb8da8bd71ed37e12aece Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Thu, 13 Oct 2022 09:22:37 +0600
Subject: [PATCH 54/92] drivers: net: lan865x: update thread and interrupt
 names

Instead of using constant strings, make those dependent on spi device
name. This makes it possible to match instances against devices in 'ps'
or 'cat /proc/interrupts' output.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 drivers/net/ethernet/microchip/lan865x/lan865x.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/microchip/lan865x/lan865x.c b/drivers/net/ethernet/microchip/lan865x/lan865x.c
index 11d25da71c48..073174a1e35e 100755
--- a/drivers/net/ethernet/microchip/lan865x/lan865x.c
+++ b/drivers/net/ethernet/microchip/lan865x/lan865x.c
@@ -852,7 +852,8 @@ static int lan865x_probe(struct spi_device *spi)
 		goto error_platform_data;
 	}
 
-	priv->tc6_task = kthread_run(tc6_handler, priv, "tc6_task");
+	priv->tc6_task = kthread_run(tc6_handler, priv, "tc6_task %s",
+				     dev_name(&spi->dev));
 	if (IS_ERR(priv->tc6_task)) {
 		ret = PTR_ERR(priv->tc6_task);
 		goto error_tc6_task;
@@ -878,7 +879,7 @@ static int lan865x_probe(struct spi_device *spi)
 
 	ret = devm_request_threaded_irq(&spi->dev, spi->irq, NULL,
 					lan865x_irq, IRQF_ONESHOT,
-					"tc6 int", priv);
+					dev_name(&spi->dev), priv);
 	if ((ret != -ENOTCONN) && (ret < 0)) {
 		dev_err(&spi->dev, "error attaching irq %d\n", ret);
 		goto error_init;
-- 
2.30.2

